cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -23,6 +23,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/addons/dns"
```

Comment:
This import chain is not allowed. Please move over the function to `constants.go` and consume in both places from there

Reply:
OK. will do :)

Reply:
>This import chain is not allowed.

BTW, does this mean that a "high-level" pkg shouldn't import a "low-level" pkg? Is this a convention of K8S or something?